plugin/vim-erlang-tags.vim

```diff
@@ -53,9 +53,13 @@ function! VimErlangTags()
     endif
 endfunction
 
-function! AsyncVimErlangTags()
+function! AsyncVimErlangTags(...)
+    let param = ""
+    for elem in a:000
+        let param = param . " " . elem
+    endfor
     let exec_cmd = s:GetExecuteCmd()
-    call system(exec_cmd . '&')
+    call system(exec_cmd . " " . param . " " . '&')
```

Comment:
It'd be nice to get the same facility in `VimErlangTags()` above too...

Reply:
@sanmiguel Hey, I made it for that function as well, but because I needed it to make the autoupdate per file work. It's all on my other —bigger— PR: https://github.com/vim-erlang/vim-erlang-tags/pull/32 😁